### PR TITLE
Name PE outputs

### PR DESCRIPTION
**Pull request by rsetaluri**

### CODE REVIEW

File: lassen/sim.py [hunk 1]
```diff
@@ -213,6 +213,7 @@ def __init__(self):
             self.rege: BitReg = BitReg()
             self.regf: BitReg = BitReg()
 
+        @name_outputs(res=Data, res_p=Bit, res_pp=Bit)
```
Comment (rsetaluri) [inline lassen/sim.py]: Not sure if these names are right, and I completely made up `res_pp`.

@alexcarsello @nikhilbhagdikar what should we name these?

Comment (leonardt) [reply lassen/sim.py]: I think the third one is `irq` (based on the return value)

**Comment by rdaly525** [comment]
This should be closed. #19 handles the name_ouputs correctly to deal with the rtl generation

**Comment by rsetaluri** [comment]
Closing in favor of #19.

**Pull request closed without merging**